contracts/monitoring/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+mod contract;
+pub mod msg;
+pub mod query;
```

Comment:
Why `pub`?

Reply:
following the convention of other contracts. Also, can not use rust files (like msg) in schema.rs if they are private.